resources/core/charts/gateway/templates/kyma-gateway-certs.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: v1
+data:
+  "key": {{ .Values.global.ingress.tlsKey }}
+  "cert": {{ .Values.global.ingress.tlsCrt }}
+kind: Secret
+metadata:
+  name: {{ .Values.global.istio.gateway.name }}-certs
+  namespace: istio-system
```

Comment:
remove namespace please

Reply:
Correction: the certs needs need to be in the same namespace that the ingress-gateway pod is